cluster-api-gcp/cloud/google/machineactuator.go

```diff
@@ -134,6 +138,7 @@ func (gce *GCEClient) CreateMachineController(cluster *clusterv1.Cluster, initia
 
 func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	config, err := gce.providerconfig(machine.Spec.ProviderConfig)
+	glog.Infof("Create.ProviderConfig = %+v", machine.Spec.ProviderConfig)
```

Comment:
Recommend putting this a a more debug level. Otherwise this may get kinda spammy.